iac-extensions/docker/src/test/resources/checks/SecretsGenerationCheck/sshKeygen.dockerfile

```
@@ -169,7 +169,13 @@ RUN ssh-keygen -N $EMPTY     -t dsa -b 1024 -f rsync-key
 RUN ssh-keygen -N $NOT_EMPTY -t dsa -b 1024 -f rsync-key
 
 # Noncompliant@+1
-RUN --mount=type=secret,id=mysecret,required RUN ssh-keygen -N $NOT_EMPTY -t dsa -b 1024 -f rsync-key
+RUN --mount=type=secret,id=mysecret,required ssh-keygen -N $NOT_EMPTY -t dsa -b 1024 -f rsync-key
+
+# Noncompliant@+2
+RUN <<EOF
+RUN ssh-keygen -N "" -b 1024 -f rsync-key -t dsa
```

Strange example, since we already are in a RUN instruction heredoc content, we should remove the second RUN.
```suggestion
ssh-keygen -N "" -b 1024 -f rsync-key -t dsa
```